### PR TITLE
RAD-388: Gradle is updated to 5.6.2. Java 11 is required

### PR DESCRIPTION
**Pull request by sa-sbertran**
Description : 
- new gradle wrapper version is 5.6.2
- Java 11 is required

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,20 +1,18 @@
...
-    dependencies {
-        classpath "org.jfrog.buildinfo:build-info-extractor-gradle:4.0.0"
-    }
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * This generated file contains a sample Java Library project to get you started.
+ * For more details take a look at the Java Libraries chapter in the Gradle
+ * User Manual available at https://docs.gradle.org/5.6.2/userguide/java_library_plugin.html
+ */
```
Comment (naraujo-sa) [inline build.gradle]: This default comment can be removed.

File: settings.gradle [hunk 2]
```diff
@@ -1 +1,10 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * The settings file is used to specify which projects to include in your build.
+ *
+ * Detailed information about configuring a multi-project build in Gradle can be found
+ * in the user manual at https://docs.gradle.org/5.6.2/userguide/multi_project_builds.html
+ */
+
```
Comment (naraujo-sa) [inline settings.gradle]: This default comment can be removed as well.

File: build.gradle (hunk 1 above)
Comment (sa-sbertran) [reply build.gradle]: 👍 removed!

File: settings.gradle (hunk 2 above)
Comment (sa-sbertran) [reply settings.gradle]: 👍 removed!

**Pull request merged**